frontend/sw360-portlet/pom.xml

```diff
@@ -111,6 +112,17 @@
             <version>${project.version}</version>
         </dependency>
 
+    <dependency>
```

Comment:
Not out of the box. It seems to be not so easy if files should be only loaded once: http://requirejs.org/docs/faq-advanced.html#css